### PR TITLE
fix(quick-add): remove flex for all components

### PR DESCRIPTION
**Pull request by SMahil**
- Revert flex changes to Patternfly grid
- Convert em/rem to px. This is the follow up work from Platfrom
- Align HTML as per guidelines http://codeguide.co/#html

***DOES NOT INCLUDE REMOVING FLEX FOR LAYOUT.  ****

### CODE REVIEW

**Comment by mindreeper2420** [comment]
@SMahil thoughts on leaving the font sizes relative to the base font size? Converting the padding, margins, widths and heights to pixels is great, but I'm concerned that if we change from the 14px font size as our base, that other typographical areas will have issues.

**Comment by hectorj2f** [comment]
[test]

File: src/app/components/planner-board/planner-board.component.scss [hunk 1]
```diff
@@ -4,28 +4,32 @@
 
 .boardWrapper {
   position: relative;
+  height: 100%;
+  overflow-y: scroll;
+  border-top: 1px solid;
+  border-top-color: #d1d1d1;
```
Comment (mindreeper2420) [inline src/app/components/planner-board/planner-board.component.scss]: set as variable `$color-pf-black-300;`

Comment (SMahil) [inline src/app/components/planner-board/planner-board.component.scss]: done

**Comment by mindreeper2420** [comment]
This PR is tied directly to PR https://github.com/fabric8io/fabric8-ui/pull/1419. Both should be merged at the same time to avoid breaking the UI.

**Comment by mindreeper2420** [comment]
@SMahil I found a few errors when I ran Planner in integrated mode that I fixed and pushed to your branch :-) The Planner will now run inside of the new Platform (no flex/Less) without issue. Everything still works in stand-alone, so I'd say this is all good @michaelkleinhenz.

File: src/app/components/iterations-modal/iterations-modal.component.scss [hunk 2]
```diff
@@ -3,27 +3,24 @@
...
-}
 .parent-iteration-container {
   position: relative;
   .down-angle-icon {
     position: absolute;
     top: 0;
     right: 0;
     background-color: $color-pf-black-400;
-    padding: em(5);
+    padding: 5px;
```
Comment (joshuawilson) [inline src/app/components/iterations-modal/iterations-modal.component.scss]: Why are we not use responsive sizes?

**Comment by joshuawilson** [comment]
The build can't find the `cico_run_tests.sh`, I think this was fixed in another PR by @michaelkleinhenz .

**Comment by joshuawilson** [comment]
@mindreeper2420 you know this does not switch to LESS, right? Can this still work when the SCSS files are not available in Platform?

File: src/app/components/iterations-modal/iterations-modal.component.scss (hunk 2 above)
Comment (SMahil) [inline src/app/components/iterations-modal/iterations-modal.component.scss]: to follow Patternfly completely, we reverted back to px as Patternfly uses px.

**Comment by mindreeper2420** [comment]
@joshuawilson I'm able to run this integrated with no problems, so not worried on the sass/less differences. The next set of work will be to switch this over to Less with a restructuring of the shared stylesheets.

File: src/app/components/iterations-modal/iterations-modal.component.scss (hunk 2 above)
Comment (joshuawilson) [inline src/app/components/iterations-modal/iterations-modal.component.scss]: ok

**Comment by joshuawilson** [comment]
@michaelkleinhenz can you please help @SMahil get this PR building as it needs to know where the `cico_run_tests.sh`.

**Comment by fabric8cd** [comment]
@SMahil snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1931-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@SMahil snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1931-fabric8-planner.int.rdu2c.fabric8.io

**Pull request merged**